frontend/src/metabase/core/components/Select/Select.jsx

```diff
@@ -206,57 +193,64 @@ export default class Select extends Component {
       .filter(n => n);
 
     return (
-      <PopoverWithTrigger
-        ref={ref => (this._popover = ref)}
-        triggerElement={
-          this.props.triggerElement || (
-            <SelectButton
-              ref={this.selectButtonRef}
-              className="flex-full"
-              hasValue={selectedNames.length > 0}
-              disabled={disabled}
-              {...buttonProps}
-            >
-              {selectedNames.length > 0
-                ? selectedNames.map((name, index) => (
-                    <span key={index}>
-                      {name}
-                      {index < selectedNames.length - 1 ? ", " : ""}
-                    </span>
-                  ))
-                : placeholder}
-            </SelectButton>
-          )
-        }
-        onClose={composeEventHandlers(onClose, this.handleClose)}
+      <TippyPopoverWithTrigger
+        placement="bottom-start"
+        onHide={onClose}
+        renderTrigger={({ onClick }) => {
+          return (
+            this.props.triggerElement || (
+              <SelectButton
+                ref={this.selectButtonRef}
+                className="flex-full"
+                hasValue={selectedNames.length > 0}
+                disabled={disabled}
+                {...buttonProps}
+                onClick={onClick}
+              >
+                {selectedNames.length > 0
+                  ? selectedNames.map((name, index) => (
+                      <span key={index}>
+                        {name}
+                        {index < selectedNames.length - 1 ? ", " : ""}
+                      </span>
+                    ))
+                  : placeholder}
+              </SelectButton>
+            )
+          );
+        }}
+        popoverContent={({ closePopover }) => (
+          <AccordionList
+            maxHeight={Infinity}
+            hasInitialFocus
+            sections={sections}
+            className="MB-Select text-brand"
+            alwaysExpanded
+            itemIsSelected={this.itemIsSelected}
+            itemIsClickable={this.itemIsClickable}
+            renderItemName={this.props.optionNameFn}
+            getItemClassName={this.props.optionClassNameFn}
+            renderItemDescription={this.props.optionDescriptionFn}
+            renderItemIcon={this.renderItemIcon}
+            onChange={option => {
+              this.handleChange(option);
+              if (!this.props.multiple) {
+                closePopover();
+              }
+            }}
```

Comment:
The only change to `AccordionList` is `onChange` prop. Since, `closePopover` is injecting via a function argument.